### PR TITLE
Add examples/notify.py: Notify of playing title

### PR DESCRIPTION
**Pull request by Matthias-Fauconneau**
I think this is a short and interesting use of playerctl.
Based on the player manager example.
Feel free to close if it is beyond the scope of an example.

### CODE REVIEW

File: examples/notify.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+from pathlib import Path
+from gi.repository import GdkPixbuf
+import os.path
+
+def notify(player):
+    metadata = player.props.metadata;
+    keys = metadata.keys()
+    if 'xesam:artist' in keys and 'xesam:title' in keys:
+        notification.update(metadata['xesam:title'], metadata['xesam:artist'][0])
+        path = Path(unquote(urlparse(metadata['xesam:url']).path)).parent / "cover.jpg";
```
Comment (acrisci) [inline examples/notify.py]: You'll need a check for `xesam:url` to be there.

And how do you know `cover.jpg` will be at that path?

**Comment by Matthias-Fauconneau** [comment]
That's a simple convention to have the album cover not only embedded but
also next to the file.
It often has that name and keeps the example simple.

**Comment by acrisci** [comment]
> That's a simple convention
How do you know that the convention will be followed? Which players are currently following the convention?

**Comment by Matthias-Fauconneau** [comment]
Here is a Python example of album art support :
https://github.com/eonpatapon/mpDris2/blob/master/src/mpDris2.in.py
When no embedded cover can be found it looks for r'^(album|cover|\.?folder|front).*\.(gif|jpeg|jpg|png)$' in related folders.

**Comment by acrisci** [comment]
:+1:

**Pull request merged**